src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
See what happens when setting `layout.title`:

http://codepen.io/etpinard/pen/LyBQLj?editors=0010

gives 

![image](https://cloud.githubusercontent.com/assets/6675409/26120301/379568b4-3a3e-11e7-9b8c-32ad2fa7185d.png)

which I believe is due to `Drawing.bBox` not computing the correct values.